### DISCUSSION TITLE
Question about y+ in PeleC

### DISCUSSION THREAD

**Discussion by Pockjob** [AMReX-Combustion/PeleC#780] [Q&A] [answered]
Hi everyone, 
How to obtain the kinematic viscosity for calculating y+ in PeleC? I have found that viscosity variable can be output in the plot files, but I am not sure if it is kinematic viscosity or dynamic viscosity. What is the unit of viscosity in PeleC ? In addition, is it  necessary to make y+ close to 1 for capturing the wall flow？

**Comment by baperry2** [comment, marked as answer]
The viscosity variable is dynamic viscosity in cgs units (used for all quantities in PeleC), so it would be g/cm/s.

Resolution requirements depend on the case and quantity of interest for the simulations, so it is up to the user to determine if their simulations are adequately resolved. In general, resolving y+ ~= 1 is often considered to be appropriate for good accuracy in wall-bounded turbulent flows, but whether that is necessary or sufficient will depend on your specific case and purposes.

**Reply by Pockjob** [reply]
Thanks for your reply. Can some wall functions be utilized in PeleC to capture wall turbulent flow? If so, y+ ~= 1 is not necessary.

**Reply by baperry2** [reply]
Wall functions are not available in PeleC and I'm not aware of any plans to add that capability. If simple enough, it may be possible to jury-rig something through the user-defined domain boundary condition (bcnormal function), but I'm not sure.

**Reply by Pockjob** [reply]
Thanks for your reply.
